src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.cpp

```diff
@@ -44,8 +46,12 @@ void CreaseDefenderFSM::blockThreat(
     }
     else
     {
-        LOG(WARNING)
-            << "Could not find a point on the defense area to block a potential shot";
+        if (event.control_params.enemy_threat_origin.x() >=
```

Comment:
`negX` implies X is negative right (so -4.5) so wouldn't all enemy threats be larger than this value?

Reply:
Yes but this would guard against any balls behind the playable field right?

Reply:
Oh I see now, makes sense. Can you add an inline comment in the code for this